jdaviz/configs/imviz/plugins/links_control/links_control.py

```diff
@@ -49,6 +90,17 @@ def __init__(self, *args, **kwargs):
                                                selected='link_type_selected',
                                                manual_options=['Pixels', 'WCS'])
 
+        self.viewer = ViewerSelect(
+            self, 'viewer_items', 'viewer_selected', 'multiselect'
+        )

```

Comment:
I think for this one you can just use the mixin and save a bit of boilerplate code,  For the layer, since you're passing `only_wcs_layers`, that might not be ideal (although probably could still use the mixin and then set `self.layer.only_wcs_layers = True` within the `__init__`.

Reply:
Good call. Updating the class to inherit from `ViewerSelectMixin`, but not `LayerSelectMixin` (for now).